std/StandardToken.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 contract StandardToken is Token {
-	uint256 public totalSupply;
```

Comment:
I'm not sure this is a desired change?

I would assume this would still work:

```
contract A {
  function totalSupply() returns (uint256);
}
contract B is A {
  uint256 public totalSupply;
}
contract C is A {
  function totalSupply() returns (uint256) {
    return 1;
  }
}
```

but this would report an error:
```
contract B is A {
  uint256 public totalSupply;
  function totalSupply() returns (uint256) {
    return 1;
  }
}
```